hathor/event/storage/event_storage.py

```diff
@@ -46,8 +46,13 @@ def iter_from_event(self, key: int) -> Iterator[BaseEvent]:
         raise NotImplementedError
 
     @abstractmethod
-    def clear_events(self) -> None:
-        """Clear all stored events and related metadata."""
+    def reset_events(self) -> None:
+        """Reset stored events and related metadata."""
+        raise NotImplementedError
+
+    @abstractmethod
+    def reset_all(self) -> None:
+        """Reset all data."""
```

Comment:
What's the difference between `reset_events()` and `reset_all()`? Maybe add some notes to each docstring such as "if you want to do XXX, you should use `reset_all()`".

Reply:
Docstrings improved in https://github.com/HathorNetwork/hathor-core/commit/79f2773a773adcbef4f5d853344e51bb66cfb4d2

Reply:
Great! Thanks!